frontend/src/storybook/StoryWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import {
+  useDatasetStoreProvider,
+  DatasetProvider,
+  startMetadataLoad,
+} from "../data/useDatasetStore";
+import Logger from "../utils/Logger";
+import { initGlobals } from "../utils/globals";
+import DataFetcher from "../data/DataFetcher";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import { ThemeProvider } from "@material-ui/styles";
+import MaterialTheme from "../styles/MaterialTheme";
+
+export const StoryWrapper = (storyFn: any) => {
+  initGlobals(new Logger(false), new DataFetcher());
```

Comment:
Is storybook only intended for local development or will it be deployed somewhere? If the former, add a TODO to make sure these dependencies are set up so they won't make real API calls.